app/controllers/users_controller.rb

```diff
@@ -1,7 +1,23 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new_anonymous_user
```

Comment:
What's the purpose of this `new_anonymous_user` method?

Reply:
I meant to remove that, it'll be gone in the next merge.